arrayadults3/arrayadults3.js

```diff
@@ -0,0 +1,15 @@
+const Person = {
```

Comment:
Als good practice for variables to write them in camelCase style.

PascalCase are reserved for Class Names, Type Names, etc... (we'll probably use them later)